zbarcam/zbarcam.py

```diff
@@ -46,6 +46,9 @@ def _setup(self):
         """
         Postpones some setup tasks that require self.ids dictionary.
         """
+        if platform == 'android':
+            from android.permissions import request_permission, Permission
+            request_permission(Permission.CAMERA)
```

Comment:
Nice PR!
Just wondering if it would make sense to `check_or_request_permission()`:
```python
@staticmethod
def check_request_permission(permission):
    """
    Android runtime permission check.
    """
    if platform != "android":
        return
    from android.permissions import (
        Permission, request_permission, check_permission)
    if not check_permission(permission):
        request_permission(permission)
```
And then:
```python
self.check_request_permission(Permission.CAMERA)
```
So basically the idea is to `check_permission()` before `request_permission()`, who it make sense?

Reply:
Maybe, but (as I tested it with c-simple without using `check_permission`) once the permission has been granted on the phone, it wont ask again for it next time you use the app.
However it mays be a good practice to check before requesting, I don't know. It's up to you ;)

Reply:
Just wanted your point of view, let's keep it your way which is simple and we would updated if needed, thanks again!